pages/api/barbershopsInfo.ts

```diff
@@ -0,0 +1,33 @@
+const handler = (
+  req: any,
+  res: {
+    status: (arg0: number) => {
```

Comment:
looks not so good yet, we will improve it together

Reply:
<img width="667" alt="Screenshot 2023-09-02 at 12 13 20" src="https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/119830208/3a257b5e-e183-420c-9242-c8bb67e58851">
<img width="667" alt="Screenshot 2023-09-02 at 12 13 05" src="https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/119830208/224b4160-c208-4a88-b0b5-e5667d260026">

Reply:
excellent!🏆